File: TP1/Hello_World/app/src/main/java/com/example/hello_world/MainActivity.java
package com.example.hello_world;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void nextAct(View v){
        Intent i = new Intent(this,MainActivity2.class);
        startActivity(i);
    }
}

File: TP1/sources/app/src/main/java/com/example/helloworld/MainActivity.kt
package com.example.helloworld

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    /**
     * Naviguate to the Hello Activity in response to button
     */
    fun naviguateToHello(view: View) {
        val intent = Intent(this, HelloActivity::class.java)
        startActivity(intent)
    }
}